integration-tests/pipelines/konflux-e2e-tests.yaml

```diff
@@ -0,0 +1,246 @@
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/konflux-qe-definitions.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: common/tasks/rosa/hosted-cp/rosa-hcp-metadata/rosa-hcp-metadata.yaml
+    - name: test-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/konflux-qe-definitions.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: common/tasks/test-metadata/0.1/test-metadata.yaml
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: test-name
+          value: $(context.pipelineRun.name)
+    - name: create-oci-container
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/konflux-qe-definitions.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: common/tasks/create-oci-artifact/0.1/create-oci-artifact.yaml
+      params:
+        - name: oci-container-repo
+          value: $(params.oci-container-repo)
+        - name: oci-container-tag
+          value: $(context.pipelineRun.name)
+    - name: provision-rosa
+      when:
+        - input: $(tasks.test-metadata.results.test-event-type)
+          operator: in
+          values: ["pull_request"]
+      runAfter:
+        - rosa-hcp-metadata
+        - test-metadata
+        - create-oci-container
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/konflux-qe-definitions.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: common/tasks/rosa/hosted-cp/rosa-hcp-provision/rosa-hcp-provision.yaml
+      params:
+        - name: cluster-name
+          value: $(tasks.rosa-hcp-metadata.results.cluster-name)
+        - name: ocp-version
+          value: $(params.ocp-version)
+        - name: replicas
+          value: $(params.replicas)
+        - name: machine-type
+          value: $(params.machine-type)
+        - name: konflux-test-infra-secret
+          value: $(params.konflux-test-infra-secret)
+        - name: cloud-credential-key
+          value: $(params.cloud-credential-key)
+    - name: konflux-e2e-tests
+      timeout: 2h
+      when:
+        - input: $(tasks.test-metadata.results.test-event-type)
+          operator: in
+          values: ["pull_request"]
+      runAfter:
+        - provision-rosa
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/e2e-tests.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: integration-tests/tasks/konflux-e2e-tests-task.yaml
```

Comment:
Integration service check PLR's result to decide if it passes or not [here](https://github.com/konflux-ci/integration-service/blob/main/helpers/integration.go#L255-L262)
Reading the task definition https://github.com/konflux-ci/e2e-tests/blob/main/integration-tests/tasks/konflux-e2e-tests-task.yaml, I can't find a `Result`,  so I am wondering how to decide if a integration pipelineRun passes, or is there any other way to check it? Thanks!

Reply:
Check integration PLR https://github.com/konflux-ci/integration-service/blob/main/internal/controller/integrationpipeline/integrationpipeline_adapter.go#L145-L149, if the PLR doesn't have a `FAIL` tekton result, integration service consider the PLR as `PASS`. 
So please ignore this comment. Thanks.